test/helpers/run-nyc.js

```diff
@@ -0,0 +1,84 @@
+const path = require('path')
+const { spawn } = require('child_process')
+
```

Comment:
I might call this `helpers.js`, because it's exporting a few other helpers other than just the `runNyc` method`.

Reply:
since it's in the helpers folder, perhaps just `index.js`?

Reply:
I've refactored the helpers dir to have an index.js which pulls in implementation from individual files.